.github/workflows/codecov.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     # cov is broken upstream, see https://github.com/rust-lang/rust/issues/79645, fix in https://github.com/rust-lang/rust/issues/93054
     if: "false"
```

Comment:
should we enable it?

Reply:
Let's try

Reply:
I suppose we need to add a `CODECOV_TOKEN` and add it to the `twitch-rs` org like described [here](https://docs.codecov.com/docs/github-2-getting-a-codecov-account-and-uploading-coverage).

Reply:
added the secret, just need to add the env to the workflow step